src/Microsoft.Health.Fhir.Core/Messages/Export/CreateExportRequest.cs

```diff
@@ -13,7 +13,16 @@ namespace Microsoft.Health.Fhir.Core.Messages.Export
 {
     public class CreateExportRequest : IRequest<CreateExportResponse>
     {
-        public CreateExportRequest(Uri requestUri, ExportJobType requestType, string resourceType = null, PartialDateTime since = null, string groupId = null, string containerName = null, string anonymizationConfigurationLocation = null, string anonymizationConfigurationFileETag = null)
+        public CreateExportRequest(
+            Uri requestUri,
+            ExportJobType requestType,
+            string resourceType = null,
+            PartialDateTime since = null,
+            string groupId = null,
+            string containerName = null,
+            string format = null,
```

Comment:
>format [](start = 19, length = 6)

If we decided to not allow format to be `null` in `ExportJobRecord` we probably shouldn't allow it be null here as well.

Reply:
Here I think it is ok for it to be null because this is what the user has passed in the _format parameter. I'll rename this to formatName to make it more clear this is not the same value.